api/src/datasets.py

```diff
@@ -353,6 +355,24 @@ def put_annotation(payload: MetadataSchema.MetaModel, dataset_id: str):
                     json=feature_payload,
                 )
 
+                feature_response = feature_response.json()
+                concept = feature["primary_ontology_id"]
+
+                if concept != "":
```

Comment:
Can `concept` be None? If so, would we not want to check?

Reply:
resolved